### ISSUE TITLE
Redirect Request: Family member hub Phase 3 (Education) - FOR Tues 9/24 launch

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#89859]
## Status
**[2024-09-17] [Fran]** Kristin says  we are good to go for all of the rows in the table that say "Final".
**[2024-09-11] [Fran]** Kristin indicates they are getting closer to launching this work. Tagged with Sprint 13. Added Sitewide tag. Here's a proposed timeline:
  - Our content team publishes the new drupal pages next Thursday (9/19)
  - Redirects are live the following Tuesday (9/24)

**[2024-08-01] [Fran]** Per Kristin, they are targeting early September, but this isn't ready to implement/release. Putting in Next Refinement, and tagging (tentatively) with Sprint 12 (Aug 21 - Sep 3).

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track different steps.
- It is your responsibility to notify VA stakeholders as appropriate.

| POC | Github alias(es) |
--- | --- 
| Content point of contact: | @d-beyer  |
| IA point of contact: | @kristinoletmuskat  |

 
### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [x] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you have a product technical team able to implement this redirect?
- [ ] Yes - Please note: Redirect still needs approval from IA.
  - [ ] Please verify your team has reviewed [Redirect governance](https://design.va.gov/components/url-standards/redirects) and [Redirect technical implementation information
](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/redirect-implementation-strategy.md) 
- [x] No - CAIA will need to make Public Websites aware of the ticket.

### Implementation date
When does this request need to be live? **NOTE:** Redirects in vsp-platform-revproxy auto-deploy M-Th. Friday launches should be avoided.
Choose one:
- [x] On a specific date, connected to a program or publishing requirement. **Tuesday, September 24**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
| Page                                               | Current URL                                                                                                                            | Redirect to                                                                                                           | Status                             |
|----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|------------------------------------|
| DEA Static page                                    | https://www.va.gov/education/survivor-dependent-benefits/dependents-education-assistance/                                              | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/                           | Final |
|                                          DEA rates | https://www.va.gov/education/benefit-rates/chapter-35-rates/                                                                           | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/rates                      | Final |
|                                   DEA future rates | https://www.va.gov/education/benefit-rates/chapter-35-rates/future-rates/                                                              | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/rates/future-rates         | Final |
|                              DEA past rates: 22-23 | https://www.va.gov/education/benefit-rates/chapter-35-rates/past-rates-2022-2023/                                                      | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/rates/past-rates-2022-2023 | Final |
|                              DEA past rates: 21-22 | https://www.va.gov/education/benefit-rates/chapter-35-rates/past-rates-2021-2022/                                                      | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/rates/past-rates-2021-2022 | Final |
|                              DEA past rates: 20-21 | https://www.va.gov/education/benefit-rates/chapter-35-rates/past-rates-2020-2021/                                                      | va.gov/family-and-caregiver-benefits/education-and-careers/dependents-education-assistance/rates/past-rates-2020-2021 | Final |
| Fry Static page                                    | https://www.va.gov/education/survivor-dependent-benefits/fry-scholarship/                                                              | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/                                           | Final |
|                                          Fry rates | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/                                                                      | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates                                      | Final |
|                                   Fry future rates | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/future-rates/                                                         | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/future-rates                         | Final |
|                              Fry past rates: 23-24 | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2023-2024/                                                 | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/past-rates-2023-2024                 |Final |
|                              Fry past rates: 22-23 | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2022-2023/                                                 | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/past-rates-2022-2023                 | Final|
|                              Fry past rates: 21-22 | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2021-2022/                                                 | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/past-rates-2021-2022                 | Final |
|                              Fry past rates: 20-21 | https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2020-2021/                                                 | va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/past-rates-2020-2021                 | Final |
| Transferred post 9/11 GI bill benefits static page | https://www.va.gov/education/survivor-dependent-benefits/transferred-benefits/                                                         | va.gov/family-and-caregiver-benefits/education-and-careers/transferred-gi-bill-benefits/                            | Final |
|                         Transferred benefits rates | https://www.va.gov/education/benefit-rates/transferred-post-9-11-gi-bill-rates/                                                        | va.gov/family-and-caregiver-benefits/education-and-careers/transferred-gi-bill-benefits/rates                        | Final |
|           Transferred benefits future future rates | https://www.va.gov/education/benefit-rates/transferred-post-9-11-gi-bill-rates/future-rates/                                           | va.gov/family-and-caregiver-benefits/education-and-careers/transferred-gi-bill-benefits/rates/future-rates           | Final |
|              Transferred benefit past rates: 23-24 | https://www.va.gov/education/benefit-rates/transferred-post-9-11-gi-bill-rates/past-rates-2023-2024/                                   | va.gov/family-and-caregiver-benefits/education-and-careers/transferred-gi-bill-benefits/rates/past-rates-2023-2024   | Final|
|              Transferred benefit past rates: 22-23 | https://www.va.gov/education/benefit-rates/transferred-post-9-11-gi-bill-rates/past-rates-2022-2023/                                   | va.gov/family-and-caregiver-benefits/education-and-careers/transferred-gi-bill-benefits/rates/past-rates-2022-2023   | Final |
|                  Education page for family members | https://www.va.gov/education/survivor-dependent-benefits/                                                                              | https://www.va.gov/family-and-caregiver-benefits/education-and-careers/                                               | Final                          |
| SECO page | https://www.va.gov/careers-employment/family-resources/                                                                           | va.gov/resources/career-resources-for-military-and-surviving-spouses                                               | Final                          |
| PCPG CH36 | https://www.va.gov/careers-employment/dependent-benefits/                                                                           | va.gov/family-and-caregiver-benefits/education-and-careers/educational-and-career-counseling                                              |            Final               | | 
 VSOC | https://www.va.gov/careers-employment/vetsuccess-on-campus/                                                                         | va.gov/resources/vetsuccess-on-campus-vsoc                                             |         Final                   |
| DEA/Fry application FORM                           | https://www.va.gov/education/apply-for-education-benefits/application/5490/introduction AND ALL CHILD PAGES                            | va.gov/family-and-caregiver-benefits/education-and-careers/apply-dea-fry-form-22-5490/                                | Final |

### Process, Roles and Responsibilities
- [x] Requesting team: Above information is provided
- [x] IA: All appropriate VA stakeholders are notified as appropriate (once we have the final URLs, can send this email after the pages have been published but before the redirect is in place)
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [x] Implementation team: Work is complete
- [x] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by kristinoletmuskat** [comment]
Hey @FranECross, we are getting closer to launching this work. Here's a proposed timeline -- would this work for you all?

- Our content team publishes the new drupal pages next Thursday (9/19)
- Redirects are live the following Tuesday (9/24)

**Comment by kristinoletmuskat** [comment]
Hey @FranECross, an update on the 2 forms in the sheet above that say "still trying to find a team" in the status column:

1. DEA/Fry Application form: we found this team led by @jsimonVA -- they are going to try to get the URL change work into the current sprint. **What is the latest deadline they need to have the URL ready to make it into our 9/24 goal?** Like does it have to be in by 9/23 at 5pm? Sooner?
2. Transferred benefit application form -- we haven't been able to find this team yet, so I'll likely be pulling this one into a separate ticket. 

Let me know if you have any Qs or thoughts!

**Comment by kristinoletmuskat** [comment]
Stakeholders have been emailed! So we are good to go for all of the rows in the table that say "Final".

**Comment by FranECross** [comment]
Excellent, @kristinoletmuskat . Thanks for the update!

**Comment by FranECross** [comment]
Per the convo in sprint planning today, I've split out the redirects not yet ready for implementation into a separate table below the main one, with a header indicating not to redirect.

**Comment by kristinoletmuskat** [comment]
Hello! Just noting that all of these static pages have been published (but not hooked up to any navs yet). So we are good to go for the redirect on Monday.

The DEA/Fry form: that team is planning to publish their changes monday morning first thing, so barring any emergencies we are good to go there too.

**Comment by randimays** [comment]
@kristinoletmuskat I saw you mentioned Monday in your last comment. The ticket ACs say we should deploy these redirects on Tuesday September 24; is that ok?

**Comment by kristinoletmuskat** [comment]
Hello @randimays yes, I just get confused about what technically happens when, but we'd like the pages to be live on the site Tuesday, so my understanding is that means you guys do something monday, right? Which thing is 'deploy'?

**Comment by jilladams** [comment]
Yes, we'll get the code changes approved and merged on Monday, and the changes will deploy / go into effect on Tuesday morning at 10a ET. ("Deploy" just means: the changes we are making get shipped out to all the servers and start being put into use.)

**Comment by randimays** [comment]
@kristinoletmuskat The redirects are in production and all are working as expected except this one:

- **Page**: Fry past rates: 20-21
- **Old URL**: https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2020-2021/
- **New URL**: https://www.va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/past-rates-2020-2021

The new URL is giving a 404; do you know if this page was published?

**Comment by kristinoletmuskat** [comment]
Hey @randimays, I will look into that one in Drupal. 

It also looks like this form might not be redirecting correctly?

va.gov/family-and-caregiver-benefits/education-and-careers/apply-dea-fry-form-22-5490/ is redirecting to https://www.va.gov/education/apply-for-education-benefits/application/5490/introduction, which is the opposite of what we want!

**Comment by randimays** [comment]
@kristinoletmuskat Great catch! I'm on it.

**Comment by kristinoletmuskat** [comment]
@randimays just checking on timing so I can coordinate menu updates -- will this deploy tomorrow am or today at some point?

**Comment by randimays** [comment]
@kristinoletmuskat I have made the fix and merged in the code; I am waiting on approval from OCTO/Platform for an out-of-band deploy (as the next production deploy is not until tomorrow morning at 10am EST). I'll post here as soon as it's fixed.

**Comment by randimays** [comment]
@kristinoletmuskat The very last redirect in the table should be working correctly in production now (I just validated). Will you check on your end please?

**Comment by kristinoletmuskat** [comment]
Yes @randimays confirmed it is working!

**Comment by randimays** [comment]
Great, thanks @kristinoletmuskat! Feel free to close the ticket when ready, or flag any other issues if they come up in your testing.

**Comment by kristinoletmuskat** [comment]
@randimays we confirmed that we forgot to publish the Fry rates 20-21 in Drupal. We're going to do that now, and I'll close the ticket once I confirm that redirect is working.

**Comment by kristinoletmuskat** [comment]
Update: there is actually internal confusion among our OCTO folks on whether we publish these old rates or archive, so this is pending until we figure that out.

**Comment by kristinoletmuskat** [comment]
Hey @randimays we've figured our stuff out, here is what we would actually like to happen:

Current page: https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2020-2021/
Redirect INSTEAD to: https://www.va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/

Can we handle this one in this ticket, or should we create a separate one?

**Comment by randimays** [comment]
@kristinoletmuskat I think it's fine to make the changes as part of this ticket. I won't be able to get this redirect in production today but I can try for tomorrow morning's deploy (it's late afternoon for me and I am pressed for time for approvals). Is there urgency on this redirect?

**Comment by kristinoletmuskat** [comment]
No supreme urgency! This is a 4-year old rates page, so we don't anticipate a lot of traffic to it anyway. But we'd love to close out this work!

**Comment by kristinoletmuskat** [comment]
Hey @randimays , just seeing when you might be able to get to this straggler

**Comment by randimays** [comment]
@kristinoletmuskat Apologies for the delay; I was on PTO Thurs-Fri last week. This straggler will be in production tomorrow morning around 10am EST 👍🏻

**Comment by randimays** [comment]
@kristinoletmuskat I re-verified all of the original redirects in the table in the description and the latest one we added:

```
Current page: https://www.va.gov/education/benefit-rates/fry-scholarship-rates/past-rates-2020-2021/
Redirect INSTEAD to: https://www.va.gov/family-and-caregiver-benefits/education-and-careers/fry-scholarship/rates/
```

Everything is working as expected. If you see any that are not working, try an incognito / private browser window or try clearing your cache. The latest one we added was a little sticky for me so I tried Chrome Incognito and it worked.

If everything looks good on your end, please close the ticket. Thank you!

**Comment by kristinoletmuskat** [comment]
You're right, it worked in incognito mode. So weird! Why does it do that?

**Comment by randimays** [comment]
@kristinoletmuskat I'm pretty sure it's a caching issue. Since that URL had a different destination previously, the browser is still holding onto that data to speed things up. Now that there's a new destination, that old data has to be shaken loose via a cache clear (when the browser session ends or whenever your browser is set to clear its own cache). Using incognito should ignore all caching/cookies so you can get a fresh call to the server to see where that page should go.
